src/components/SearchBar.tsx

```diff
@@ -64,19 +66,21 @@ const SearchAppBar = ({ back }: SearchProps) => {
               <SearchIcon />
             </div>
             <InputBase
-              placeholder="peepeepoopoo"
+              placeholder="searchie search"
```

Comment:
Can this placeholder be "Search for an article or resource" or whatever Fang has in the designs?